src/strategies/spidStrategy.ts

```diff
@@ -83,7 +83,9 @@ const spidStrategy = (
         privateCert: samlKey
       }
     },
-    (profile, done) => done(undefined, profile)
+    // tslint:disable-next-line:no-any
```

Comment:
disabling `no-any` should be a last resort for very special cases - in this case I'm sure you'll be able to define more specific types